cpp/src/svm/workingset.cuh

```diff
@@ -252,10 +256,11 @@ class WorkingSet {
    *     DOI: 10.1080/10556780500140714
    *
    * @param [in] alpha device vector of dual coefficients, size [n_train]
-   * @param [in] C penalty parameter
+   * @param [in] C_vec penalty parameter
    * @param [in] nc number of elements to select
    */
-  int PrioritySelect(math_t *alpha, math_t C, int nc) {
+  int PrioritySelect(math_t *alpha, const math_t *C_vec, int nc) {
+    math_t C = 1;  // TODO remove this and use C_vec
```

Comment:
I'm guessing you'll fix it here? Because it seems that the rest of this function is still using `C`

Reply:
Yes, I fixed this. (I was skipping this initially because this working set selection strategy is not used currently #946).